addons/l10n_ec/models/res_partner.py

```diff
@@ -69,22 +69,26 @@ def check_vat(self):
                     if partner.l10n_latam_identification_type_id.id == it_ruc.id and len(partner.vat) != 13:
                         raise ValidationError(_('If your identification type is %s, it must be 13 digits')
                                               % it_ruc.display_name)
-                    final_consumer = verify_final_consumer(partner.vat)
```

Comment:
@jco-odoo no need, this is here just to bypass the warning message for 9999999999999 and thus it is not required in base_vat. BTW in base_vat if someone wants to register the final consumer it will be recorded with no problem because the length is 13.